src/components/EmojiPicker/EmojiPicker.tsx

```diff
@@ -181,14 +181,14 @@ const EmojiPicker = forwardRef((props, ref) => {
                 vertical: emojiPopoverAnchorPosition.vertical,
                 horizontal: emojiPopoverAnchorPosition.horizontal,
             }}
-            anchorRef={getEmojiPopoverAnchor()}
+            anchorRef={getEmojiPopoverAnchor() as RefObject<View | HTMLDivElement>}
```

Comment:
Instead of assertion let's try adjusting anchorRef type OR getEmojiPopoverAnchor() return type

Reply:
I'm okay with this assertion then, mostly because this PR is already quite big. We will handle it once whole TS migration is finished. Thank you!